tests/test_data/sweetpeasandsaffron.com/sweetpeasandsaffron_1.json

```diff
@@ -0,0 +1,63 @@
+{
+  "author": "Denise Bustard",
+  "canonical_url": "https://sweetpeasandsaffron.com/vegan-carrot-ginger-soup-instant-pot/",
+  "site_name": "Sweet Peas and Saffron",
+  "host": "sweetpeasandsaffron.com",
+  "language": "en-US",
+  "title": "Instant Pot Carrot Ginger Soup (Vegan)",
+  "ingredients": [
+    "1 tablespoon olive oil",
+    "1 onion ((chopped))",
+    "2 cloves garlic ((minced))",
+    "2 tablespoons ginger ((grated or finely chopped))",
+    "5 cups carrots ((peeled & chopped))",
+    "4 cups vegetable broth",
+    "3/4 teaspoons salt",
+    "1/2 teaspoon pepper",
+    "1 teaspoon dried thyme leaves",
+    "13.5 oz coconut milk ((400 mL; full fat works best))",
+    "1/2 lime ((juiced))"
+  ],
+  "instructions_list": [
+    "Select the sauté function of the Instant Pot. Add olive oil and onion to the inner pot. Cook for 5-6 minutes, until onions are softened and translucent.",
+    "Add the garlic & ginger, cook 1-2 more minutes. Turn off sauté function.",
+    "Add carrots, broth, salt and pepper and thyme leaves to the inner pot. Stir to combine. Put the lid on, turn vent to 'sealed' and cook on manual high pressure for 5 minutes.",
+    "After the timer beeps, allow the pot to sit for ten minutes (natural pressure release), before releasing the remaining pressure.",
+    "Blend until smooth with an immersion blender, or carefully transfer to a stand blender and puree in batches.",
+    "Stir in coconut milk and lime juice, and enjoy!",
+    "Stove Top",
+    "Cook onion in olive oil over medium heat for 5-6 minutes.",
+    "Add the garlic & ginger, cook 1-2 more minutes.",
+    "Add carrots, stock, salt, pepper and thyme, and stir to combine. Cook for 20-30 minutes, until carrots are soft.",
+    "Blend until smooth with an immersion blender.",
+    "Stir in coconut milk and lime juice, and enjoy!"
+  ],
+  "category": "Soup",
+  "yields": "4 servings",
```

Comment:
A small detail here: the literal `recipeYields` value in the source is 4-6 -- we return only the 4 (min/first) part of that.

The nutritional info is listed as per-sixth of a batch.  So in a way it would be nice to have consistency between the yield count and the nutritional serving size.

Reply:
I agree, I'm not seeing a clear way to gather this info though.. I supposed we could check if a range exists formatted like `num 1 - num 2` and grab the larger of the range?

This page contains: `"recipeYield":["4","4 -6"]`

Reply:
Apologies for the slow reply; I've been thinking about this too but agree that it's non-obvious how to resolve it.

The best idea I have so far is: perhaps we could run some kind of check to examine how often the `recipeYields` value in schema contains a dash character between two numbers and/or contains a list/array value.  That would help us figure out how frequently this happens -- and from there, whether we should support something at the `yields` field level, or perhaps something specific only to this scraper.

Arguably given that yields can be somewhat variable, we should go with the former option.. but even that wouldn't guarantee a strong link between the nutrition servings and the recipe servings (and that may be OK, since `schema.org` [`NutritionInfo`](https://schema.org/NutritionInformation) does have its own `servingSize` field anyway, and it seems theoretically valid -- if sometimes confusing -- for that to differ from the recipe description).

Reply:
(I don't think this is a blocker for merge; I'll re-review in a few moments)